.github/PULL_REQUEST_TEMPLATE/manual_maintenance.md

```diff
@@ -0,0 +1,67 @@
+# Manual Maintenance Sprint [NR]
+# Maintenance tasks (to be completed by the assignee)
+## Skyminder
+- [ ] Complete manual maintenance in the Skyminder Repo
+- [ ] Release a new version `x.x.x` (Please put new version number here)
+## EDC
+- [ ] Complete manual maintenance in the EDC Repo
+- [ ] Release a new version `x.x.x` (Please put new version number here)
+## Dataland
+### Gradle update
+- [ ] Execute `gradlew dependencyUpdates` to get a report on Dependencies with updates
+- [ ] Update `settings.gradle.kts` (for libraries), `build.gradle.kts` (for plugins) and `gradle.properties` (for jacoco and ktlint)
+Note: fasterXML is managed by spring and ktlint by jlleitschuh, thus NO manual version update should be conducted
+- [ ] update the gradle wrapper: execute `gradle wrapper --gradle-version X.Y.Z`
+
+### Dataland frontend
+- [ ] Update node version in `dataland-frontend/build.gradle.kts`
+- [ ] Update node packages: run the `updatepackages` script, e.g. by  `npm run updatepackages` to update versions in package.json
+- [ ]   Run the `updatepackagelock`, e.g. by  `npm run updatepackagelock` script to update `package-lock.json` and check for security issues
```

Comment:
wenn ich `npm run updatepackagelock` ausführe verändert sich meine package-lock

Reply:
Vielleicht liegt das daran, dass node/npm in der aktuellen version andere keys in die package-lock schreiben. Ich nutze node 18.7.0 - gebundled mit npm 8.15.0

Reply:
![image](https://user-images.githubusercontent.com/47553592/186409184-441542df-27c3-4e8e-aadb-d89fb9bca86c.png)

Reply:
ok